File: JavaScript/1550_Three_Consecutive_Odds/solution2.js
/**
 * @param {number[]} arr
 * @return {boolean}
 */
const threeConsecutiveOdds = function (arr) {
    for (let i = 0; i < arr.length - 2; i++) {
        if ((arr[i] * arr[i + 1] * arr[i + 2]) % 2 == 1) {
            return true;
        }
    }
    return false;
};

File: Go/1550_Three_Consecutive_Odds/solution2.go
package solution

func threeConsecutiveOdds(arr []int) bool {
	counter := 0
	for i := 0; i < len(arr); i++ {
		if arr[i]%2 == 1 {
			counter++
		} else {
			counter = 0
		}

		if counter > 2 {
			return true
		}
	}

	return false
}
